### ISSUE TITLE
Update README on https://hub.docker.com/r/jupyter/repo2docker

### ISSUE THREAD

**Issue by manics** [jupyterhub/repo2docker#1076]
If someone has push rights to https://hub.docker.com/r/jupyter/repo2docker I think it'd be good to:
- Manually push `quay.io/jupyterhub/repo2docker:2021.08.0` and a `latest` tag to Docker Hub
- Manually update the docker Hub readme to point to quay.io

**Comment by choldgraf** [comment]
Can either @yuvipanda or @minrk or @consideRatio either add @manics to the `jupyterhubbinder` team on DockerHub, or update the README per the above to say that it is deprecated? I just tried but apparently do not have admin privileges myself (https://hub.docker.com/orgs/jupyter)

**Comment by yuvipanda** [comment]
<img width="468" alt="image" src="https://user-images.githubusercontent.com/30430/133557607-e6079396-4468-48f4-a0a8-db29ad8feb34.png">

Looks like we need to pay dockerhub to make any changes other than removal to this group of people...

**Comment by yuvipanda** [comment]
<img width="788" alt="image" src="https://user-images.githubusercontent.com/30430/133557722-95bc2f18-3867-4103-aaac-e2335d298a6d.png">

**Comment by minrk** [comment]
I pushed the tags and updated the readme. Bummer that we can't really use DockerHub anymore, but I certainly understand getting crushed by offering a wildly successful free service.

IIRC, we got in touch with docker about their open source plans, but I don't think anyone heard back. Or did we?

**Comment by manics** [comment]
Thanks for updating the readme!

I don't think you were in last months team meeting where we decide to move repo2docker to quay.io. Part of the reason was a lack of confidence in Docker Hub's long term viability given Docker's funding issues. However this is probably worth a team discussion... I'll add it to today's meeting agenda.

**Comment by minrk** [comment]
Yeah, I totally get it. I don't mean to imply I'm unhappy with the decision. Fully support shifting to quay.io, especially given that we can't even give ourselves enough permissions to update the README!
